### PR TITLE
Add json export app UI

### PR DESCRIPTION
**Pull request by MrCreosote**

### CODE REVIEW

File: ui/narrative/methods/export_json_to_staging/spec.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      "allow_multiple" : false,
+      "default_values" : [ "workspace_export" ],
+      "field_type" : "text"
+    },
+    {
+      "id": "format",
+      "optional" : false,
+      "advanced" : false,
+      "allow_multiple" : false,
+      "default_values" : [ "legacy_data_import_export" ],
```
Comment (Tianhao-Gu) [inline ui/narrative/methods/export_json_to_staging/spec.json:36]: we want default to be `legacy` not `standard`?

**Review by Tianhao-Gu — APPROVED** [review]
👍 other than question regarding default format.

File: ui/narrative/methods/export_json_to_staging/spec.json (hunk 1 above)
Comment (MrCreosote) [reply ui/narrative/methods/export_json_to_staging/spec.json:36]: Legacy due to the principle of least astonishment

File: ui/narrative/methods/export_json_to_staging/display.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        ui-name: |
+            Destination Directory
+        short-hint : |
+            The destination directory to which the file will be saved.
+
+    format:
+        ui-name: |
+            Output format
+        short-hint : |
+            The format of the output zip file. standard is the output as returned from the workspace. legacy_data_import_export is the format that was returned from direct JSON downloads via the app panel prior to the coversion to this app.
```
Comment (briehl) [inline ui/narrative/methods/export_json_to_staging/display.yaml]: typo: "coversion" to this app -> "conversion" to this app

File: ui/narrative/methods/export_json_to_staging/spec.json [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  "parameters" : [
+    {
+      "id" : "input_ref",
+      "optional" : false,
+      "advanced" : false,
+      "allow_multiple" : false,
+      "default_values" : [ "" ],
+      "field_type" : "text",
+      "text_options" : {
+        "valid_ws_types" : ["*"]
```
Comment (briehl) [inline ui/narrative/methods/export_json_to_staging/spec.json:20]: Noting here that this won't do much and look like an error until https://kbase-jira.atlassian.net/browse/UIP-48 is done.

File: ui/narrative/methods/export_json_to_staging/display.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+description : |
+    <p>This App is designed to export the JSON representation of a KBase object to the Staging Area.</p>
+
+    <p>The App allows the user to select a folder in the Staging Area where the files will be located. This can be done by setting the folder name in the parameter Destination Directory. If the folder named in Destination Directory does not exist, a new folder will be created. For convenience, the workspace_export folder name is filled in by default as a place to manage files for download. The Staging Area can be accessed via the Data Panel by clicking the + icon; this action will slide out the data panel. The Destination Directory can be found in the Import tab. Once the selected directory is accessed, files can be downloaded by clicking the downwards arrow.</p>
```
Comment (briehl) [inline ui/narrative/methods/export_json_to_staging/display.yaml:44]: Might want to add some text here about what name the object will have. I think the existing downloader gives `object_name.JSON.zip` or something like that. I'm assuming it'll be the same.

Also mention what'll happen if a file with that name already exists - overwrites or errors.

**Review by briehl — CHANGES_REQUESTED** [review]
A typo and some doc text request.

File: ui/narrative/methods/export_json_to_staging/spec.json (hunk 3 above)
Comment (MrCreosote) [reply ui/narrative/methods/export_json_to_staging/spec.json:20]: yep, understood

File: ui/narrative/methods/export_json_to_staging/display.yaml (hunk 4 above)
Comment (MrCreosote) [reply ui/narrative/methods/export_json_to_staging/display.yaml:44]: Added

File: ui/narrative/methods/export_json_to_staging/display.yaml (hunk 2 above)
Comment (MrCreosote) [reply ui/narrative/methods/export_json_to_staging/display.yaml]: Fixed

**Review by briehl — APPROVED** [review]
Looks good.

**Pull request merged**